src/framework/standard/mod.rs

```diff
@@ -260,8 +261,12 @@ impl StandardFramework {
                     return Some(DispatchError::BlockedGuild);
                 }
 
-                if let Some(guild) = guild_id.to_guild_cached(&ctx.cache).await {
-                    if self.config.blocked_users.contains(&guild.owner_id) {
+                let cache = ctx.cache.clone();
+                let guild_selector = |guild: &Guild| guild.owner_id;
+                let owner_id_option = cache.guild_field(guild_id, guild_selector).await;
```

Comment:
I don't see the usefulness of placing the closure into its own variable here, it can just be inline with the `guild_field` call. The type of the parameter can also be omitted; the compiler will infer it. Furthermore, I also don't understand why the cache has to be cloned here. It is wrapped in an `Arc` for sure, which is cheap to clone, but nonetheless, it could be nice to avoid that.
```suggestion
                let owner_id_option = ctx.cache.guild_field(guild_id, |guild| guild.owner_id).await;
```

Reply:
I have no idea why I cloned the `Arc` either. I'll change that. As for the separating of the closures, I'll inline them for this one and also all the other cases I did it. Generally, I like to separate closures from what they're being used for when they're really long because I think it's slightly cleaner. In this case with the owner ID, I have no idea why I did it though either :/